packages/patternfly-4/react-core/src/components/LoginPage/LoginForm.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import { Form, FormGroup, ActionGroup, FormHelperText } from '../Form';
+import { TextInput } from '../TextInput';
+import { Button } from '../Button';
+import { Checkbox } from '../Checkbox';
+
+export interface LoginFormProps extends React.HTMLProps<HTMLFormElement> {
+  /** Additional classes added to the Login Main Body's Form */
+  className?: string;
+  /** Flag indicating the Helper Text is visible * */
+  showHelperText?: boolean;
+  /** Content displayed in the Helper Text component * */
+  helperText?: React.ReactNode;
+  /** Label for the Username Input Field */
+  usernameLabel?: string;
+  /** Value for the Username */
+  usernameValue?: string;
+  /** Function that handles the onChange event for the Username */
+  onChangeUsername?(value: string, event: React.FormEvent<HTMLInputElement>): void;
```

Comment:
Change the function types to arrow functions like `onChangeUsername?: (value: string, event: React.FormEvent<HTMLInputElement>) => void`